5.6/test/run

```diff
@@ -51,7 +51,8 @@ prepare() {
 }
 
 run_test_application() {
-  run_args=${CONTAINER_ARGS:-}
+  run_args="${CONTAINER_ARGS:-} -e PHP_OPTIONS=memory_limit=32M"
```

Comment:
the whole point of the CONTAINER_ARGS env variable is that you set it when you invoke run_test_application, not override it within run_test_application.